joomla/joomla-cms
# [4.0] Remove libraries/cms/html/language/en-GB/en-GB.jhtmldate.ini

### Summary of Changes
This PR removes the language file libraries/cms/html/language/en-GB/en-GB.jhtmldate.ini.
The file isn't in a regular language folder where it could be loaded, and the strings in it are also present in the lib_joomla.ini one.


### Testing Instructions
Check the Users -> Users Action Logs view
The column "Date" uses the strings in that file.

### Expected result
![image](https://user-images.githubusercontent.com/1018684/51661696-eac59300-1fb1-11e9-8d81-032f1c09f4f4.png)



### Actual result
Same as expected


### Documentation Changes Required
None
